src/coreclr/src/tools/Common/TypeSystem/Common/MethodDesc.ToString.cs

```diff
@@ -46,7 +46,9 @@ public override string ToString()
         {
             var sb = new StringBuilder();
 
-            // (Skipping return type to keep things short)
+            sb.Append(Signature.ReturnType);
```

Comment:
Ok I can revert that. I was hoping this would provide better diagnostics, not just for covariant return methods, but also for cases where we could have methods with the same name/sig, but different return types.